apps/editing-toolkit/editing-toolkit-plugin/wpcom-block-editor-nux/class-wp-rest-wpcom-block-editor-nux-status-controller.php

```diff
@@ -69,10 +69,6 @@ public function is_nux_enabled( $nux_status ) {
 	 * @return WP_REST_Response
 	 */
 	public function get_nux_status() {
-		// Check if we want to show the Welcome Tour variant pbxNRc-Cb-p2
-		// Performing the check here means that we'll only assign a user to an experiment when this API is first called.
-		$welcome_tour_show_variant = ( defined( 'SHOW_WELCOME_TOUR' ) && SHOW_WELCOME_TOUR ) || apply_filters( 'a8c_enable_wpcom_welcome_tour', false );
```

Comment:
> Thank you for this reminder!! We do not want to show on mobile. Ola is working on mobile designs, til then we show on desktop / larger screens only. Thus, we will need to identify mobile and prevent showing. 

Good to know. 

Once option might be to keep the filter here temporarily, since the mobile check is on the backend. 

Then, in the backend filter, we would remove the code related to the experiment.

Or we move the mobile check to the ETK (and remove the WPCOM filter entirely) so we have consistent behaviour across Simple and Atomic sites. What do you think?

Reply:
Thanks for those tips!  I moved the check to ETK: https://github.com/Automattic/wp-calypso/pull/51055/commits/95cd894f9ac0fe4119a724102b232592728435c2

Reply:
We will show the NUX modal to mobile users.